resources/views/pages/home.blade.php

```diff
@@ -1,8 +1,188 @@
 <x-general-layout>
-    <div class="container mx-auto py-5 prose mb-32">
-        <span class="text-eph-purple font-bold">Online fysiotherapie</span>
-        <h1 class="heading-h1">Waar en wanneer het jou uitkomt</h1>
-        <p class="text-eph-grey">Behandeld worden door professionele fysiotherapeuten door heel Nederland.</p>
-        <x-links.button class="btn-primary rounded-full mt-4" href="#">Hoe werkt het?</x-link-button>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css">
+
+
+    <!--Hero section1-->
+    <section class="container m-auto px-6">
+        <div class="lg:flex justify-between items-center">
+            <div class="lg:w-6/12 lg:p-0 p-7">
+                <small class="text-eph-purple font-bold">Online fysiotherapie</small>
+                <h1 class="heading-h1">Waar en wanneer het jou uitkomt</h1>
+                <p class="text-eph-grey mt-6">Behandeld worden door professionele fysiotherapeuten door heel Nederland.</p>
+                <div class="py-5">
+                    <x-links.button class="btn-primary rounded-full mt-4 font-normal" href="#">Hoe werkt het?</x-link-button>
+                </div>
+            </div>
+              <div class="lg:w-5/12">
+                <img class="rounded" src="https://images.unsplash.com/photo-1517245386807-bb43f82c33c4?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1350&q=80">
+              </div>
+        </div>
+    </section>
+
+
+
```

Comment:
Enters zijn prima, maar één keer wat witruimte is genoeg. ;)

Reply:
Hoe bedoel je dat hier? zit minimaal 1 witregel tussen?